### ISSUE TITLE
Macros to hide rows/columns, change row height/column width don't work

### ISSUE THREAD

**Issue by hope-full** [ONLYOFFICE/DesktopEditors#313]
Steps to reproduce:
1.Open a spreadsheet document
2.Set
`Api.GetActiveSheet().GetRows(__your_rows_here__).SetHidden(true);` or
`Api.GetActiveSheet().GetCols(__your_rows_here__).SetHidden(true);` or
`Api.GetActiveSheet().GetRows(__your_rows_here__).SetRowHeight(__your_value_here__);` or
`Api.GetActiveSheet().GetRows(__your_rows_here__).SetColumnWidth(__your_value_here__);`
to some rows.

Actual result:
The macros work on some empty rows/columns (especially on a new blank document), but do nothing if the columns/rows have some content

Expected result:
The macros should hide/resize rows/columns

**Comment by hope-full** [comment]
Changed GetActiveWorksheet() to GetActiveSheet().
Sorry 'bout that.

**Comment by flaminestone** [comment]
Hi @hope-full. I can confirm - it is bug, №45277 in our bugtraker. We will fix it in future, all news will be in this issue

**Comment by FlorentWillemin** [comment]
Hello, any news about that?

**Comment by ShockwaveNN** [comment]
@FlorentWillemin Sorry, currently there is no news, I've asked our dev team if they got any ETA for fixing it

**Comment by TheChafoin** [comment]
Hi, the problem is still present, is there any update ?

**Comment by askonev** [comment]
@TheChafoin @hope-full 

Script for hiding rows and columns
```js
var oWorksheet = Api.GetActiveSheet();
// var oRange = oWorksheet.GetCols("A:C")
var oRange = oWorksheet.GetRows("1:3")
oRange.SetHidden(true);
```

The script for setting the width and height
>Fixed on version 7.2.0
```js
var oWorksheet = Api.GetActiveSheet();
var oRange = oWorksheet.GetRange("A1:C3");
    oRange.SetRowHeight(32);
    oRange.SetColumnWidth(32);
```

**Comment by Rita-Bubnova** [comment]
I close this issue. Feel free to comment or reopen it if you got further questions.
